packages/opub-ui/src/components/Breadcrumbs/Breadcrumbs.module.scss

```diff
@@ -0,0 +1,80 @@
+@import '../../../styles/common';
+
+.Breadcrumbs {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 4rem;
```

Comment:
dont use spacing value that are not in tokens.
In this case, don't provide a margin-top. We don't know in which context this breadcrumb component will be used